src/App.tsx

```diff
@@ -7,13 +7,29 @@ import SettingsMenu from "./SettingsMenu";
 import { SettingOutlined } from "@ant-design/icons";
 import Heatran from "./Heatran";
 import { ChangeLog } from "./ChangeLog";
+import { ThemeSwitcherProvider } from "react-css-theme-switcher";
+import { getCookie } from 'cookied';
+
 const logo = require("./assets/pgricon64.png");
 
+const currThemes = {
+  dark: '/modomator/dark-theme.css',
+  light: '/modomator/light-theme.css'
+};
+
+const currentTheme = getCookie('currentTheme');
```

Comment:
Related to another comment: Is there a reason to cookies over LocalStorage here?

Reply:
no, changed to LocalStorage in next commit